docs/source/getting_started/content/ref_dns-configuration.adoc

```diff
@@ -1,6 +1,19 @@
 [id="dns-configuration_{context}"]
 = DNS configuration details
 
+[id="dns-configuration-general{context}"]
+== General DNS setup
+
+The OpenShift cluster managed by {prod} uses 2 DNS domain names, `crc.testing` and `apps-crc.testing`.
+`crc.testing` is used for core OpenShift services.
+`apps-crc.testing` is used when accessing OpenShift applications deployed on the cluster.
+
+For example, the OpenShift API server will be exposed as `api.crc.testing` while the OpenShift console is accessed through `console-openshift-console.apps-crc.testing`
+These DNS domains are served by a `dnsmasq` DNS container running inside the {prod} virtual machine.
+
+Running [command]`{bin} setup` will adjust your system DNS configuration so that it can resolve these domains.
+When [command] `{bin} start` is run, additional checks will be done to verify DNS is properly configured.
```

Comment:
`When [command] `{bin} start`  runs,` ?

Reply:
I don't know which form is better, I'll wait for kevin's comment before updating the PR.

Reply:
I'd reverse the clauses and use active voice:

```adoc
Additional checks are done to verify DNS is properly configured when running [command]`{bin} start`.
```